.github/workflows/tests.yaml

```diff
@@ -0,0 +1,82 @@
+name: Tests
+
+on: [ push ]
+
+env:
+    COMPOSER_ROOT_VERSION: "dev-master"
+
+jobs:
+    build-matrix:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.4
+                    coverage: none
+                env:
+                    COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            -   run: composer install --no-progress --ansi
+            -   run: vendor/bin/monorepo-builder validate
+            -   id: set-packages
+                run: echo "::set-output name=packages::$(vendor/bin/monorepo-builder packages-json)"
+        outputs:
+            packages: ${{ steps.set-packages.outputs.packages }}
+
+    tests:
+        needs: build-matrix
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                symfony-version:
+                    - '4.4.*'
+                    - '5.2.*'
+                composer-lowest:
+                    - 'no'
+                    - 'yes'
```

Comment:
Given the size of the matrix, maybe we should just always use `lowest`?

Reply:
I think we should also always test agains the latest versions.

Reply:
I updated the exclude to only run once